d2renderers/d2svg/d2svg.go

```diff
@@ -224,6 +230,39 @@ func arrowheadMarker(isTarget bool, id string, connection d2target.Connection) s
 				width*0.6, height*7/8,
 			)
 		}
+	case d2target.FilledCircleArrowhead:
+		attrs := fmt.Sprintf(`class="connection" fill="%s" stroke-width="%d"`, connection.Stroke, connection.StrokeWidth)
+		offset := 4.0 + float64(connection.StrokeWidth*2)
+		if isTarget {
+			path = fmt.Sprintf(`<circle %s cx="%f" cy="%f" r="%f"/>`,
+				attrs,
+				offset+9, height/2,
+				offset*1.2,
+			)
+		} else {
+			path = fmt.Sprintf(`<circle %s cx="%f" cy="%f" r="%f"/>`,
+				attrs,
+				offset+3, height/2,
+				offset*1.2,
+			)
+		}
+	case d2target.CircleArrowhead:
+		attrs := fmt.Sprintf(`class="connection" fill="white" stroke="%s" stroke-width="%d"`, connection.Stroke, connection.StrokeWidth)
```

Comment:
can this be transparent instead of white?

Reply:
https://github.com/terrastruct/d2/issues/635

Reply:
> can this be transparent instead of white?

https://github.com/terrastruct/d2/issues/635